src/tests/stress/savina/5.fjcreate/ForkJoinConfig.enc

```diff
@@ -0,0 +1,21 @@
+module ForkJoinConfig
+
+read class ForkJoinConfig
+  val N: int
+  val C: int
+
+  def init(N: int, C: int): unit
+    this.N = N
+    this.C = C
+  end
+
+  def performComputation(theta: real): unit
+    val sint = EMBED (real)
+                 sin(#{theta});
+               END
+    val res = sint * sint
+    if (res <= 0) then
```

Comment:
I didn't make this up :)
[Original code here](https://github.com/shamsimam/savina/blob/master/src/main/java/edu/rice/habanero/benchmarks/fjcreate/ForkJoinConfig.java#L47)

```scala
protected static void performComputation(final double theta) {
    final double sint = Math.sin(theta);
    final double res = sint * sint;
    //defeat dead code elimination
    if (res <= 0) {
        throw new IllegalStateException("Benchmark exited with unrealistic res value " + res);
    }
}
```